lib/git-cmd.js

```diff
@@ -347,4 +363,29 @@ export default {
 				return files;
 			});
 	},
+
+	/**
+	 * Get repo branches
+	 * @param {string} cwd Current Working Directory
+	 * @return {Promise} {object[]} Repository branch names
+	 */
+	branches(cwd) {
+		return this.cmd(cwd, ["branch", "--list", "-a"])
+		.then((result) => {
+			if(result === ""){
+				return []
+			}
+
+			const branches = result.trimRight().split("\n").map((line) => {
+				let path = line.trimRight().trimLeft();
+				let name = path.replace(/^\* /, '').replace(/^\/remotes\/origin/, '');
```

Comment:
I'm assuming the remote is named 'origin' here.  This was done to support branches with the slash character (e.g. /feature/25).  If you think that will be a problem I could try to find a way to identify the remote name instead.